ui/src/app/flags/variants/Variants.tsx

```diff
@@ -10,24 +10,26 @@ import Button from '~/components/forms/buttons/Button';
 import Modal from '~/components/Modal';
 import DeletePanel from '~/components/panels/DeletePanel';
 import Slideover from '~/components/Slideover';
-import { deleteVariant } from '~/data/api';
+import { useAppDispatch } from '~/data/hooks/store';
 import { IFlag } from '~/types/Flag';
 import { IVariant } from '~/types/Variant';
+import { deleteVariantAsync } from '../flagsSlice';
```

Comment:
```suggestion
import { deleteVariantAsync } from '~/app/flags/flagsSlice';
```

Reply:
might have to run prettier again afterwards too `mage ui:fmt` from root, or `npx prettier -w .` from `ui` dir